doc/api/url.md

```diff
@@ -948,6 +948,20 @@ changes:
 The `url.format()` method returns a formatted URL string derived from
 `urlObject`.
 
+```js
+url.format({
+  protocol: 'https',
+  hostname: `example.com`,
```

Comment:
Are the backticks intended? If so, are not they a bit confusing here? With them, a template literal/a tag function/other quotes inside are usually expected.

Reply:
Good catch, @vsemozhetbyt. Backticks were not intentional there. Fixed.